test/metabase/sync/analyze/fingerprint/insights_test.clj

```diff
@@ -80,3 +80,59 @@
 (expect
   false
   (valid-period? #inst "2015-01" #inst "2015-02" nil))
+
+
+;; Test for https://github.com/metabase/metabase/issues/9070
```

Comment:
Can you add a note about what this is testing, so people don't have to go to GitHub to work it out